mappings/net/minecraft/client/gui/screen/ingame/SignEditScreen.mapping

```diff
@@ -1,6 +1,10 @@
 CLASS net/minecraft/class_498 net/minecraft/client/gui/screen/ingame/SignEditScreen
+	FIELD field_21525 model Lnet/minecraft/class_837$class_4702;
+	FIELD field_24285 text [Ljava/lang/String;
```

Comment:
```suggestion
	FIELD field_24285 textLines [Ljava/lang/String;
```

Reply:
Should `SignBlockEntity.text` also be `textLines` then?

https://github.com/FabricMC/yarn/blob/5642461bdeb9cdd118718bca7101cb891c33ccf5/mappings/net/minecraft/block/entity/SignBlockEntity.mapping#L5

Reply:
Yeah probably